resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolver.java

```diff
@@ -213,7 +218,6 @@ public DnsNameResolver(
             int ndots,
             boolean decodeIdn) {
         super(eventLoop);
-        checkNotNull(channelFactory, "channelFactory");
```

Comment:
I can revert this if you prefer ... its not necessary for this PR. Just noticed it during development.